### PR TITLE
feat: add terminal color check

### PR DESCRIPTION
**Pull request by hengyoush**
add terminal color check

### CODE REVIEW

File: common/term.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import (
+	"os"
+	"strings"
+)
+
+func Is256ColorSupported() bool {
+	term := os.Getenv("TERM")
+	return strings.Contains(term, "256color")
+}
```
Comment (ccoVeille) [inline common/term.go:11]: I'm sorry but this check is way too naive

Take a look at project that could help you to detect that capabilities

https://pkg.go.dev/github.com/gookit/color#IsSupport256Color

**Review by ccoVeille — CHANGES_REQUESTED** [review]

File: common/term.go (hunk 1 above)
Comment (ccoVeille) [reply common/term.go:11]: The simplest would be to use this 

https://pkg.go.dev/github.com/charmbracelet/lipgloss#DefaultRenderer

The call ColorProfile

Comment (hengyoush) [reply common/term.go:11]: Thank you for pointing out the issues! I modify this part of the code later.

Comment (ccoVeille) [reply common/term.go:11]: I opened
 - #https://github.com/hengyoush/kyanos/issues/186

**Pull request merged**